rules/helm/delete.tmpl.rb

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env ruby
+
+# frozen_string_literal: true
+
+require 'open3'
+
+# Variable interpolation via Bazel template expansion.
+helm = '[[helm]]'
+install_name = '[[install_name]]'
+namespace = '[[namespace]]'
+
+stdout, stderr, status = Open3.capture3(
```

Comment:
This eats `stderr` on success…

```suggestion
exit Proess.wait2(Process.spawn(helm, 'delete', install_name, '--namespace', namespace)).last.exitstatus
```

Reply:
I'll accept this, but I'll do it in a separate commit as I need to remove the `open3` requirement.

Reply:
That's fine… I actually rarely look at the per-commit level, and just look at the PR as-is.